cvmfs/network/download.cc

```diff
@@ -290,6 +281,11 @@ static size_t CallbackCurlData(void *ptr, size_t size, size_t nmemb,
   const size_t num_bytes = size*nmemb;
   JobInfo *info = static_cast<JobInfo *>(info_link);
 
+  // TODO(heretherebedragons) remove if no error comes up
+  // as this means only jobinfo data request (and not header only)
+  // come here
+  assert(info->sink != NULL);
+
```

Comment:
This assert can be removed?

Reply:
i am not sure.. i added it because it was not clear to us if this is always true. probably it is..

Reply:
Ok, then let's leave it.